build_tools/github_actions/configure_ci.py

```diff
@@ -179,7 +179,10 @@ def get_benchmark_presets(trailers: Mapping[str, str]) -> str:
 def main():
   output: MutableMapping[str, str] = {}
   event_name = os.environ["GITHUB_EVENT_NAME"]
-  trailers = get_trailers() if event_name == PULL_REQUEST_EVENT_NAME else {}
+  trailers = get_trailers() if event_name == PULL_REQUEST_EVENT_NAME else {
+      # Run all benchmarks if it isn't a pull request event.
+      BENCHMARK_PRESET_KEY: "all"
+  }
```

Comment:
How about instead passing `event_name` to `get_benchmark_presets`, so the `trailers` variable is not mutated?

```python
def get_benchmark_presets(...):
  if event_name != PULL_REQUEST_EVENT_NAME:
    preset_options = list(
        option for option in BENCHMARK_PRESET_OPTIONS if option != "all")
    ...
    
trailers = get_trailers() ...
output["benchmark-presets"] = get_benchmark_presets(trailers, event_name)
```

As this is, the logging will be inaccurate
```python
  print(f"Using benchmark preset '{trailer}' from PR description trailers")
```

Reply:
SG. Done.